plugin-server/functional_tests/ingestion.test.ts

```diff
@@ -104,6 +104,37 @@ test.concurrent(`event ingestion: can set and update group properties`, async ()
     })
 })
 
+test.concurrent(`event ingestion: handles $groupidentify with no properties`, async () => {
+    const teamId = await createTeam(postgres, organizationId)
+    const distinctId = new UUIDT().toString()
+
+    const groupIdentityUuid = new UUIDT().toString()
+    await capture(producer, teamId, distinctId, groupIdentityUuid, '$groupidentify', {
+        distinct_id: distinctId,
+        $group_type: 'organization',
+        $group_key: 'posthog',
```

Comment:
nit: maybe `_test` in the end. There are times, when test code ends up in prod and the name being searchable would be useful

Reply:
hah yeah. So that shouldn't be an issue really with these as we create a new team that is associated with them. I could update the team name to make it obvious, I think it's just a uuid or something like that atm.